cli/cmd/simulation/temp.hpp

```diff
@@ -198,7 +202,29 @@ class temp_command : public command
      * Critical temperature statistics.
      */
     fiction::critical_temperature_stats stats{};
+    /**
+     * The simulation engine to use.
+     */
+    std::string sim_engine_str{"QuickExact"};
+    /**
+     * Convert the simulation engine string to the appropriate engine enum member. QuickExact is set as default.
+     *
+     * @return The `sidb_simulation_engine` member associated with the identifier.
+     */
+    [[nodiscard]] inline fiction::sidb_simulation_engine get_sim_engine() const noexcept
+    {
+        if (sim_engine_str == "ClusterComplete")
+        {
+            return fiction::sidb_simulation_engine::CLUSTERCOMPLETE;
+        }
 
+        if (sim_engine_str == "QuickSim")
+        {
+            return fiction::sidb_simulation_engine::QUICKSIM;
+        }
+
+        return fiction::sidb_simulation_engine::QUICKEXACT;
+    }
```

Comment:
As mentioned above, this function could be replaced with a global one selecting engines.

Reply:
good suggestion, done